.nvmrc

```diff
@@ -0,0 +1 @@
+6.10.0
```

Comment:
The file is used by the build script `travis-build` generates. Unless you need this file when `travis-build` runs and compiles the bash script, this is not necessary. (Notice that `.nvmrc` does not currently exist for Node.js builds, either.)